tests/test_job.py

```diff
@@ -70,12 +70,10 @@ def setUp(self, request):
         self._tmp_dir = TemporaryDirectory(prefix="signac_")
         request.addfinalizer(self._tmp_dir.cleanup)
         self._tmp_pr = os.path.join(self._tmp_dir.name, "pr")
-        self._tmp_wd = os.path.join(self._tmp_dir.name, "wd")
+        self._tmp_wd = os.path.join(self._tmp_dir.name, "pr", "workspace")
```

Comment:
At first I thought this was unused -- it's only used in `test_project.py` which has tests inheriting from this class, and it's only used twice. Maybe we can consolidate that somehow.

Reply:
I just removed one of the places, which wasn't really testing anything useful, and hardcoded the path in the one test that verifies that the workspace directory is set to the expected path.